hack/generator/pkg/codegen/pipeline_export_generated_code.go

```diff
@@ -87,33 +88,58 @@ func writeFiles(ctx context.Context, packages map[astmodel.PackageReference]*ast
 	globalProgress := newProgressMeter()
 	groupProgress := newProgressMeter()
 
+	var wg sync.WaitGroup
```

Comment:
How do you limit having too many of these running at once?

If there are too many, I'd expect output congestion, probably need to limit it to #CPUs or something like that.

Reply:
Limited to 8; it's going to be IO bound as well so just going by CPU count isn't super useful. 8 is a nice round number :D